src/DetailsView/components/settings-panel/settings/bug-filing/bug-filing-settings.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import * as React from 'react';
+import { FlaggedComponent } from '../../../../../common/components/flagged-component';
+import { FeatureFlags } from '../../../../../common/feature-flags';
+import { NamedSFC } from '../../../../../common/react/named-sfc';
+import { SettingsProps } from '../settings-props';
+
+export const BugFilingSettings = NamedSFC<SettingsProps>('BugFilingSettings', props => {
+    const getBugSettingsUx = () => {
+        return (
+            <>
+                <h3>Issue filing</h3>
+                {getGitHubBugSettingsUx()}
+            </>
+        );
+    };
+
+    const getGitHubBugSettingsUx = () => {
+        return (
+            <TextField
+                className="issue-setting"
+                label="Enter desired GitHub repo link:"
+                onChange={onGitHubRepositoryChange}
+                value={getBugServiceProperty('gitHub', 'repository')}
+                placeholder="https://github.com/owner/repo/issues"
```

Comment:
I thought the change was made where we dont really need to put in `issues` as the url. So, will it make sense to update `placeholder` text accordingly?

Reply:
talked offline. resolve this.